s2i/src/main/docker/project/pom.xml

```diff
@@ -1,21 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```

Comment:
Erhm, I didn't do this. Guessing we should be adding this to `.gitignore` or something?